en/ai/ai-providers-and-api-keys.md

```diff
@@ -0,0 +1,102 @@
+# AI providers and API keys
+
+## What is an AI provider?
+
+AI provider is a company or a service that gives you the ability to send requests to and receive responses from LLM. In order to get the response, you also need to send an API key to authenticate and manage billing.
+
+Here is the list of AI providers we currently support: OpenAI, Mistral AI, Hugging Face. Others include Google Vertex AI, Microsoft Azure OpenAI, Anthropic, etc.
```

Comment:
What does "etc." mean? Either the list is complete or incomplete. If it is incomplete, state where one could get the information about the complete list.

Maybe, they can be distinguished between checked and possibly working. --> Think about the Privacy policy. We need to list **all** providers which JabRef directly supports. For others, users need to check the privacy policy for themselves.

Reply:
I provided link to this page https://docs.langchain4j.dev/category/language-models

What do you think, is it okay?